src/component/variantPage/therapeuticImplication/OncoKb.tsx

```diff
@@ -0,0 +1,219 @@
+import * as React from 'react';
+import { observer } from 'mobx-react';
+import _ from 'lodash';
+// TODO remove the IndicatorQueryTreatment import after exposed in cbioportal-frontend-commons
+import {
+    IndicatorQueryResp,
+    IndicatorQueryTreatment,
+} from 'oncokb-ts-api-client';
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+import { generateOncokbLink, ONCOKB_URL } from '../biologicalFunction/Oncokb';
+
+import functionalGroupsStyle from '../functionalGroups.module.scss';
+import DrugList from '../DrugList';
+
+interface IOncoKbProps {
+    oncokb: IndicatorQueryResp | undefined;
+    isCanonicalTranscriptSelected: boolean;
+}
+
+export const SENSITIVE_LEVELS = [
+    'LEVEL_1',
+    'LEVEL_2',
+    'LEVEL_3',
+    'LEVEL_3A',
+    'LEVEL_3B',
+    'LEVEL_4',
+];
+
+export const RESISTANT_LEVELS = ['LEVEL_R1', 'LEVEL_R2'];
+
+@observer
+class OncoKb extends React.Component<IOncoKbProps> {
```

Comment:
The code here is not new. Moved from `src/component/variantPage/TherapeuticImplication.tsx`.